store/navigation.js

```diff
@@ -0,0 +1,38 @@
+import contentfulClient from '../plugins/contentful.js';
+
+export const state = () => ({
+  data: []
+});
+
+export const mutations = {
+  setData(state, data) {
+    state.data = data;
+  }
+};
+
+export const actions = {
+  async init({ commit }) {
+    const i18n = this.app.context.app.i18n;
+    function isoLookUp(code) {
+      const locales = i18n.locales;
+      return locales.find(locale => locale.code === code)['iso'];
+    }
+
+    const setLocale = i18n.locale;
+    const data = await contentfulClient.getEntries({
+      'locale': isoLookUp(setLocale),
+      'content_type': 'linkGroup',
+      'fields.identifier': 'pageMainNavigation'
```

Comment:
what do you think about using`mainNavigiation` and updating the identifier in contentful?

I think page is redundant and adds more confusion with the issue Mirjam Raised above re pagination & navigation components